### ISSUE TITLE
Sell giant pedestal grinder

### ISSUE THREAD

**Issue by andrewadare** [SolidStateDepot/ssd-ops#4]
This monster is sitting in the weld shop. It requires 440V 3-phase power, which we don't currently have.

**Comment by jtadodd** [comment]
The board decided that we’d be giving this up as it’s not something that’s in line with the use of the space.

**Comment by andrewadare** [comment]
Next step is to find the right facebook marketplace account to use for the sale.

**Comment by jtadodd** [comment]
True, I couldn’t see how to as SSD, but I can list as me for it if
necessary
